### PR TITLE
Feature/settings page

### PR DESCRIPTION
**Pull request by DavidKeanu**

### CODE REVIEW

File: src/layouts/BaseLayout.tsx [hunk 1]
```diff
@@ -61,6 +62,11 @@ export const BaseLayout: FC = () => {
                       <ArchivePage />
                     </Route>
 
+                    {/* Archive page */}
+                    <Route exact path="/settings">
```
Comment (hstaatsxx) [inline src/layouts/BaseLayout.tsx:66]: Ich würde Settings ganz unten packen :)

**Comment by DavidKeanu** [comment]
Update: Moved settings page down in the router component.

**Comment by hstaatsxx** [comment]
Settings goes to bottom of side menu.
Changed wording for each language edition.
- German edition
- Englische Ausgabe

**Comment by DevJacobi** [comment]
Testing was done by @hstaatsxx but not documented because of sickness. To keep our documentation consistent I added it now:

https://user-images.githubusercontent.com/43578928/205732297-33bf3c95-311e-4c98-bebe-666914b013bd.mp4

**Pull request merged**